contrib/macdeploy/README.md

```diff
@@ -1,15 +1,135 @@
-### MacDeploy ###
+# Mac Deployment
 
-For Snow Leopard (which uses [Python 2.6](http://www.python.org/download/releases/2.6/)), you will need the param_parser package:
+The `macdeployqtplus` script should not be run manually. Instead, after building as usual:
 
-	sudo easy_install argparse
+```bash
+make deploy
+```
 
-This script should not be run manually, instead, after building as usual:
+During the deployment process, the disk image window will pop up briefly
+when the fancy settings are applied. This is normal, please do not interfere,
+the process will unmount the DMG and cleanup before finishing.
 
-	make deploy
+When complete, it will have produced `Bitcoin-Qt.dmg`.
 
-During the process, the disk image window will pop up briefly where the fancy
-settings are applied. This is normal, please do not interfere.
+## SDK Extraction
 
-When finished, it will produce `Bitcoin-Qt.dmg`.
+After Xcode version 7.x, Apple started shipping the `Xcode.app` in a `.xip` archive.
```

Comment:
Nit: we can lose the historical reference and just say "`Xcode.app` is packaged in a `.xip` archive.